### PR TITLE
handle favorites

### PR DESCRIPTION
**Pull request by Anna812**
- create endpoint handling in retrofit for favorites
- make feedfragment use parameter to handle feed and favorite feed
- update pageradapter

### CODE REVIEW

**Comment by adamgyulavari** [comment]
@Anna812 resolve conflicts

**Comment by Anna812** [comment]
conflicts resolved :)

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/FeedFragment.java [hunk 1]
```diff
@@ -71,6 +84,16 @@ public void onFailure(Call<ArrayList<NewsItem>> call, Throwable t) {
             }
         });
     }
+
+    public static FeedFragment newInstance(int someInt) {
+        FeedFragment myFragment = new FeedFragment();
+
+        Bundle args = new Bundle();
+        args.putInt("someInt", someInt);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/FeedFragment.java]: @Anna812 can we name this meaningful instead "someInt". eg: a boolean named onlyFavorites

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/FeedFragment.java (hunk 1 above)
Comment (Anna812) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/FeedFragment.java]: changed to tabNumber as it should indicate which tab are you on, maybe we will add a third one just for fun :)

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**